NeoMathEngine/src/CPU/x86/avx/src/xbyak/xbyak.h

```diff
@@ -75,6 +75,9 @@
 	#ifndef WIN32_LEAN_AND_MEAN
 		#define WIN32_LEAN_AND_MEAN
 	#endif
+	#ifndef NOMINMAX
```

Comment:
This is a third-party code. Are you __absolutely__ sure that you need to change it?

Reply:
NOMINMAX define replaced from library to upper level include file